include/oneapi/dpl/pstl/tuple_impl.h

```diff
@@ -542,19 +619,31 @@ operator-(Size idx, const oneapi::dpl::__internal::tuple<T1...>& tuple1)
 namespace std
 {
 template <size_t _Idx, typename... _Tp>
-constexpr typename oneapi::dpl::__internal::tuple_element<_Idx, oneapi::dpl::__internal::tuple<_Tp...>>::type&
+constexpr typename ::std::tuple_element<_Idx, oneapi::dpl::__internal::tuple<_Tp...>>::type&
 get(oneapi::dpl::__internal::tuple<_Tp...>& __a)
 {
-    return oneapi::dpl::__internal::get_impl<_Idx>()(__a);
+    return __a.template get<_Idx>();
```

Comment:
What is the difference between current and removed line? Just call the `get()` method of `__a` directly? I see compilation failures on Ubuntu20 for `g++` compiler.

Reply:
there is not big difference. I just want that structured binding goes the same path as just std::get. The goal is to avoid duality in the implementation for the conceptually same functionality

Reply:
I see. However, there are some failures now. Could you please investigate?

Reply:
I believe the only reason for the failure is `constexpr` in c++11. I was able to reproduce it locally. All tests should pass after add conditionally `constexpr` since C++14 but if it's not the case we will see how to deal with that.